README.md

```diff
@@ -1,39 +1,41 @@
-# News challenge
+# News Summary Challenge
 
-* Feel free to use Google, your notes, books, etc. but work on your own.
-* If you refer to the solution of another coach or student, please put a link to that in
-  your README.
-* If you have a partial solution, **still check in a partial solution** and send in a pull
-  request.
-* You must submit a pull request to this repo with your code by 10am Monday morning.
+This is a single web page that shows the latest news using the Guardian API. The aim of the challenge is to practice writing code in frontend JavaScript, CSS and HTML.
 
-## Challenge
+## Description
 
-As usual please start by forking this repo.
+The app grabs all the headlines from the Guardian newspaper API and display them on a
+page. Clicking on a headline links the user to the actual article page on the Guardian's website.
 
-You'll create an app that shows the latest news using the Guardian API. You'll find
-instructions on how to setup and use the API down this page.
+## How the webpage looks like
 
-## Project overview
+![screenshot](https://github.com/valentina-maggio/news-summary/blob/main/public/images/Screenshot%201.png?raw=true)
+![screenshot2](https://github.com/valentina-maggio/news-summary/blob/main/public/images/Screenshot%202.png?raw=true)
 
-Your app will grab all the headlines from the Guardian newspaper API and display them on a
-page. Clicking on a headline will link the user to the actual article page on the Guardian's website.
+## How to run the app locally
 
-### Technologies
+* Clone this repo
+* Install Node (and nvm if not previously installed)
+* Move to the project directory
+* Run npm install to install all dependencies
+* Install Esbuild with `npm install -g esbuild`
+* Run the command `npm run build`
+* Type `open index.html` in the command line to visit the we page
 
-You'll write a single page web app. You'll write your code in frontend JavaScript, CSS and
-HTML. You won't use Ruby or backend JavaScript.
+### Note on API credentials
 
-You're strongly encouraged to use the same tools and libraries you've been learning this
-week:
- * Jest to write your tests
- * The [`jest-fetch-mock`](https://www.npmjs.com/package/jest-fetch-mock) module (or
-   similar) to mock `fetch` requests (so you don't request the API when running tests!)
- * A build tool such as `esbuild` to bundle files together
+The credentials are not uploaded in the current folder. Please use the guardianApiKeyTemplate.js to save your API key and follow the instructions saved in the file.
```

Comment:
This is good enough. Given that this projects is front end, the user would have access to the API key anyway, but at least it's not on Github. What you can do to when using an API key on the front-end is tell the API provider which domains it should authorise requests from, so at least people are not copying the API key into their own projects.

Reply:
Good tip! Thank you :)